DragonSlayer.cpp

```diff
@@ -1,21 +1,34 @@
 #include "DragonSlayer.h"
 #include "Dragon.h"
+#include "Utility.h"
 
-//DragonSlayer::DragonSlayer
+DragonSlayer::DragonSlayer(std::string name_, int hitPoints_, int armor_) : Character(hitPoints_, armor_, 4), name(name_), attackItem(new AttackItem())
```

Comment:
You can initialize your attack item directly where it is declared.
You don't need to initialize it in the Member Initializer List
- [ ] revise